api/src/main/java/com/discordsrv/api/discord/entity/message/DiscordMessageEmbed.java

```diff
@@ -284,6 +285,17 @@ public Builder setColor(int rgb) {
             return setColor(new Color(rgb));
         }
 
+        @NotNull
+        public Builder setColor(@Nullable CharSequence hex) {
+            try {
+                return setColor(new Color(hex != null ? hex.toString() : null));
+            } catch (IllegalArgumentException e) {
+                // Either the hex is invalid or it's a placeholder that still haven't been processed
+                this.color = null;
+                return this;
+            }
+        }
+
```

Comment:
This alone won't let placeholders be parsed, the color needs to be stored in a new field like `unformattedColor`, handled in:
- `SendableDiscordMessageImpl$FormatterImpl` (converting unformatted to the actual value and setting it), and
- `DiscordMessageEmbedSerializer` (to store info the unformatted field if the input is otherwise a "invalid" String)

and same for timestamp

Also passing null to the color constructor throws NPE